core/src/test/resources/fielderrorregion/Foo.java

```diff
@@ -0,0 +1,29 @@
+package test;
+
+import javax.annotation.Nullable;
+
+public class Foo {
+  Bar f0 = new Bar().process(null);
+  Bar f1;
+  Bar f2, f3;
+  @Nullable Bar nullableBar;
+  Bar f4 = nullableBar.process(new Object());
```

Comment:
New test case looks great to me! Could we still add `Bar f5;` without initialization or dereference to this single constructor test case and make sure it's marked as `@Nullable`. Just for the sake of testing all cases. If I understood the explanation correctly, then I fully expect it to work and then we can land this 🙂

Reply:
Sure, please find it 970a2abe3aa4ff841faaf515e59c00c68d3ad602